Zombie/ZombieDriver.js

```diff
@@ -1,10 +1,10 @@
 "use strict";
 
-process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
-
 const Zombie = require('zombie');
 const zombie = new Zombie();
 
+process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
```

Comment:
Good point, we can't do it after the return... but I think in the sample above `process.on` will still be called before `new Bridge` as that function will only be called after it's returned to .NET.

You'll need to place it inside the function, before we hand code back to .NET by calling the callback:
```
//  Entry point for Edge.js
return function(data, callback) {
    let bridge = new Bridge(zombie);
    process.on('uncaughtException', e => console.error('Uncaught Exception: ' + e));
    callback(null, bridge);
}
```